src/vs/workbench/contrib/search/common/searchModel.ts

```diff
@@ -157,6 +157,10 @@ export class Match {
 	getMatchString(): string {
 		return this._oneLinePreviewText.substring(this._rangeInPreviewText.startColumn - 1, this._rangeInPreviewText.endColumn - 1);
 	}
+
+	toJSON(): object {
+		return {}; // We send an IRenderableMatchContext to the extensions
```

Comment:
Such changes aren't good and data that's transfers between processes should have explicit DTOs (data transfer objects)